src/io/flutter/actions/DeviceSelectorAction.java

```diff
@@ -0,0 +1,47 @@
+package io.flutter.actions;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.DefaultActionGroup;
+import com.intellij.openapi.actionSystem.ex.ComboBoxAction;
+import com.intellij.openapi.project.DumbAware;
+import icons.FlutterIcons;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+
+
+public class DeviceSelectorAction extends ComboBoxAction implements DumbAware {
+
+    @Override
+    public void actionPerformed(AnActionEvent e) {
+        //TODO: fill in.
```

Comment:
curios, how is this `actionPerformed` different from the one below? Which gets called when the switch devices?